Doc/whatsnew/3.11.rst

```diff
@@ -62,6 +62,7 @@ Summary -- Release highlights
 .. This section singles out the most important changes in Python 3.11.
    Brevity is key.
 
+- Python 3.11 is 22% faster than Python 3.10. See `Faster CPython`_ for details.
```

Comment:
Maybe add something like "on the standard benchmark suite"? Just so people don't get upset if whatever they're doing doesn't get faster.